docs/environments/simnet.md

```diff
@@ -67,12 +67,13 @@ The following is a command reference to get going:
 
   `$ dcrd --simnet --rpcuser=<username> --rpcpass=<password> --miningaddr=<S....>`
 
-* Instruct dcrd to generate enough initial blocks for the first coinbase to mature:
+* Instruct dcrd to generate enough initial blocks for the first coinbase to mature. Generate genesis block, first mining block with coinbase then more 16 for confirmations:
```

Comment:
Can we please replace these two sentences with just one:

> Instruct dcrd to generate the genesis block, followed by 16 more blocks for the first coinbase to mature.


Also I guess the number below can be updated to 17 rather than 18?

Reply:
[genesis block](https://github.com/decred/dcrd/blob/4da12163dcbe0e36c52bca89d22325101666f49d/chaincfg/simnetparams.go#L67) don't send coins to anyone, then just the second block creates valid coinbase

Reply:
Interesting! I wasn't aware of this. Lets add that detail to the guide to explain why we need 18 blocks rather than 17.

Reply:
I think we can merge this PR and work on best simnet explain, I'm started with #61 maybe I can change for simnet page improvements and add all points to explain the simnet envr